scripts/compare_benchmarks.py

```diff
@@ -210,8 +210,8 @@ def double_vertical_separators(lines, vertical_separators_to_duplicate):
     table_data.append(
         [
             name,
-            f"{(old_avg_successful_duration / 1e6):>7.1f}" if old_avg_successful_duration else "nan",
-            f"{(new_avg_successful_duration / 1e6):>7.1f}" if new_avg_successful_duration else "nan",
+            f"{(old_avg_successful_duration / 1e6):>3.4f}" if old_avg_successful_duration else "nan",
+            f"{(new_avg_successful_duration / 1e6):>3.4f}" if new_avg_successful_duration else "nan",
```

Comment:
At least in the PR description, the border of the column looks broken:

```
+----------++----------+---------+---------++----------+----------+---------+---------+
| TPC-H 01 ||    1.4635 |  2.4467 |  +67%  ||     0.68 |    19.81 | +2800%  |  0.0000 |
```

Reply:
I think something went wrong by copying it in to the PR description. If I execute the script inside my terminal everything works fine. 

<img width="778" alt="Screenshot 2021-01-03 at 11 45 12" src="https://user-images.githubusercontent.com/32880898/103476815-26318480-4db9-11eb-9a08-1ce309603ee3.png">